app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::CategoriesController < ApiController
+
+  def index
+    categories = JAPI::Trebek.categories(options = {count: 4, offset: rand(10000)})
+    
+    clues = [] 
+    categories.each do |category|
+      clues << category.clues
+    end
+
+    render json: {
+      categories: categories,
+      clues: clues
+    }
+  end
+
+  # def categories_response(categories_data_array)
+  #   categories_with_clues = []
+
+  #   categories_data_array.each do |category|
+  #     categories_with_clues << JAPI::Trebek.category(category)
+  #   end
+
+  #   # do I need to clean up the categories_with_clues array, or is it ready to be returend to the frontend
+
+  #     # categories_with_clues
+  # end
+
+  # def parsed_category(category_with_clues)
+
+  #   cleaned_category = {
+  #     category: { 
+  #       title: categories_with_clues.title,
+  #       clues: categories_with_clues.questions
+  #     }
+  #   }
+  # end
+end
```

Comment:
wait, so based on your current implementation, you WERE able to get your clues from the initital categories request? if so, great, you can remove the commentd out private methods. Otherwise this feature is unfinished?